### PR TITLE
pytest anyio

### PR DESCRIPTION
**Pull request by aminalaee**
Add pytest anyio

### CODE REVIEW

**Review by graingert — CHANGES_REQUESTED** [review]
I think it would be best to use `@pytest.mark.anyio`

**Comment by aminalaee** [comment]
@graingert Thanks. There's no `anyio` in `orm`. Do you mean we add that too?

**Comment by graingert** [comment]
afaik, orm and databases are designed with Starlette in mind and so should also work with anyio eventually

**Comment by aminalaee** [comment]
Screenshot from 2021-08-20 13-09-51
My guess is your PR in `databases` anyio port should go first.

File: setup.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def get_packages(package):
     packages=get_packages(PACKAGE),
     package_data={PACKAGE: ["py.typed"]},
     data_files=[("", ["LICENSE.md"])],
-    install_requires=["databases>=0.2.1", "typesystem"],
+    install_requires=["anyio>=3.0.0,<4", "databases>=0.2.1", "typesystem"],
```
Comment (graingert) [inline setup.py:53]:
```suggestion
    install_requires=["databases>=0.2.1", "typesystem"],
```

File: setup.py [hunk 2]
```diff
@@ -50,7 +50,7 @@ def get_packages(package):
...
     extras_require={
         "postgresql": ["asyncpg"],
```
Comment (graingert) [inline setup.py:55]:
```suggestion
        "tests": ["anyio>=3.3.0,<4"],
        "postgresql": ["asyncpg"],
```

**Review by graingert — CHANGES_REQUESTED** [review]
anyio is only a test dep for now

File: setup.py (hunk 2 above)
Comment (graingert) [reply setup.py:55]: sorry this suggestion is wrong test deps go here

https://github.com/encode/orm/blob/master/requirements.txt

Comment (aminalaee) [reply setup.py:55]: @graingert  we can add anyio to the `requirements.txt` instead of extra requirements.

**Pull request merged**